cdist/conf/type/__package_yum/gencode-remote

```diff
@@ -60,10 +60,10 @@ fi
 
 case "$state_should" in
     present)
-        echo yum $opts install \"$install_name\"
+        echo "yum $opts install \"$install_name\""
```

Comment:
We could switch the inner `\"` to `'` here too.

Reply:
fixed, was due to be fixed yesterday evening, but somehow...